pysmt/solvers/msat.py

```diff
@@ -48,6 +47,88 @@
 from pysmt.walkers.identitydag import IdentityDagWalker
 
 
+class _MSatEnvWrapper(object):
```

Comment:
Minor: I think this can just be called ```MSatEnvWrapper``` without the leading underscore.
Due to the ref-counting handling, this class becomes the preferred method to handle the mathsat environment at the python level.